ooniprobe/View/TestInfoViewController.mm

```diff
@@ -53,7 +53,9 @@ -(void)reloadTest{
 
 -(IBAction)runTest:(id)sender{
     NetworkMeasurement *current = [currentTests getTestWithName:testName];
-    current.inputs = nil;
+    //TODO change this code when refactor NetworkMeasurement class
+    [current setUri_scheme:NO];
```

Comment:
I am not sure I really like the name of this variable. Why is it named like this?

Reply:
uri_scheme is needed to disable max_runtime. do you have any name suggestion?